mirrord/operator/src/client.rs

```diff
@@ -145,7 +150,22 @@ impl OperatorApi {
             }
             version_progress.success(None);
 
+            let client_certificate =
+                if let Some(credential_name) = status.spec.license.fingerprint.as_ref() {
+                    CredentialStoreSync::with_exclusive_lock::<MirrordOperatorCrd, _, _>(
+                        &operator_api.client,
+                        credential_name.to_string(),
+                        |credentails| Ok(credentails.as_ref().clone()),
+                    )
+                    .await
+                    .map_err(|err| debug!("CredentialStore error: {err}"))
```

Comment:
Is `debug` the level we want here (asking as I don't actually know)?

Reply:
I don't think this is something the user should care about. Mostly for us